sdk/nodejs/runtime/stack.ts

```diff
@@ -67,7 +70,7 @@ class Stack extends ComponentResource<Inputs> {
      *
      * @param init The callback to run in the context of this Pulumi stack
      */
-    async initialize(args: { init: () => Promise<Inputs> }): Promise<Inputs> {
+    async initialize(): Promise<Inputs> {
```

Comment:
Is this safe to change?

Reply:
I'm honestly unsure _why_ this changed. I'll try going without it.

Reply:
Reverted.